### DISCUSSION TITLE
not compatible with switchresx?

### DISCUSSION THREAD

**Discussion by Joostjez** [MonitorControl/MonitorControl#1556] [General]
### Before opening the issue, have you...?

- [x] Searched for existing issues

### Question

Well: general Q:  monitor control is not compatible with switchresex?

**Comment by rmohns** [comment]
Works fine here. I only use SwitchResX to enable some HiDPI modes on non-Apple displays, nothing fancy. But it does not conflict with MonitorControl that I can tell.

**Comment by waydabber** [comment]
Yes, both MonitorControl and https://github.com/waydabber/BetterDisplay are compatible with SwitchResX.
